src/VirtualClient/VirtualClient.Dependencies/CUDAAndNvidiaGPUDriverInstallation.cs

```diff
@@ -156,13 +208,23 @@ await this.stateManager.SaveStateAsync(nameof(CudaAndNvidiaGPUDriverInstallation
 
                     SystemManagement.IsRebootRequested = true;
                 }
+                else if (this.Platform == PlatformID.Win32NT)
+                {
+                    await this.CudaAndNvidiaGPUDriverInstallationOnWindowsAsync(telemetryContext, cancellationToken)
+                               .ConfigureAwait(false);
+
+                    await this.stateManager.SaveStateAsync(nameof(this.CudaAndNvidiaGPUDriverInstallationOnWindowsAsync), new State(), cancellationToken)
+                        .ConfigureAwait(false);
+
+                    SystemManagement.IsRebootRequested = this.RebootRequired;
+                }
                 else
                 {
                     // CUDA and Nvidia driver installation for other platforms to be added.
                     throw new WorkloadException(
```

Comment:
Just take this part out, right now under .NET 6 there is only Unix and Windows

Reply:
Modified as suggested.